docs/pages/introduction/versioning.md

```diff
@@ -0,0 +1,18 @@
+---
+category: Introduction
+page: Versioning policy
+---
+
+# Base UI Versioning Policy
+
+We recognize that you need stability from the Base UI.
+
+This document describes the practices that we follow to provide you the Base UI library. We want everyone who depends on Base UI to know when and how new features are added, and to be well-prepared when obsolete ones are removed.
+
+Development builds of Base UI include many helpful warnings. Whenever possible, we add warnings in preparation for future breaking changes. These warnings do not affect the production version of your application.
+
+We are following [SemVer](https://semver.org), with a few flavours. We do not bump major versions for the following changes:
+
+- Any component or functions that's prefixed with `Unstable_` / `unstable_` may change or got removed without notice,
```

Comment:
Just so i'm clear on what you're thinking, i would import an unstable component like below?

```js
import {Unstable_FileUploader} from 'baseui/file-uploader';
```

Reply:
yes, I think so

or should we just go for `UnstableFileUploader`?

Reply:
`Unstable_FileUploader` might be more obvious when scrolling through the codebase.